spotify/main.py

```
@@ -183,6 +183,7 @@ def main():
         open_podcast_client,
         start,
         end,
+        fallible=True,
```

I would suggest something more informative such as `continueOnError` or `doNotRaiseErrors`. Even though I would aso suggest to log the error and not just silently return. Can be a just q quick logline like "404 on URL..."

---

Good idea. Renamed to `continueOnError`.

---

Good idea. Renamed to `continueOnError`.